common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -1534,6 +1534,24 @@
     ],
     "sqlState" : "42601"
   },
+  "IDENTITY_COLUMNS_DUPLICATED_DESCRIPTOR" : {
+    "message" : [
+      "Duplicated IDENTITY column descriptor: <descriptor>."
```

Comment:
What's a "descriptor"? Are we exposing interals to SQL here?

Reply:
No, we just show whether START WITH option or INCREMENT BY are duplicated here. Changed the name to `sequenceGeneratorOption` here.